go.mod

```diff
@@ -1,14 +1,16 @@
 module terraform-provider-maas
 
-go 1.20
```

Comment:
Could you please revert this change? go version bump + toolchain

Reply:
Hm, it seems that this change was here due to https://github.com/canonical/terraform-provider-maas/pull/179
> go: github.com/bflad/tfproviderlint@v0.30.0 requires go@1.21, but 1.20 is requested